src/Storefront/Framework/SystemCheck/SaleChannelsReadinessCheck.php

```diff
@@ -20,6 +20,8 @@
 
 /**
  * @internal
+ *
+ * @codeCoverageIgnore covered with integration tests/integration/Storefront/Framework/HealthCheck/SaleChannelsReadinessCheckTest.php
```

Comment:
oh, @nfortier-shopware changed annotations like this in another PR. the `codeCoverageIgnore` seems to need to be alone on a line